demos/mailer/README.md

```diff
@@ -22,5 +21,8 @@ unclassified:
 It is identified as YAML root element `mailer` as this descriptor has a `global.jelly` UI view, so JCasC
 assumes it makes sense to expose it as a root element extension.
 
-Descriptor defines setters so we can inject configuration, but for SMTP authentication parameters.
-See [jenknisci/mailer-plugin#39](https://github.com/jenkinsci/mailer-plugin/pull/39)
+~~Descriptor defines setters so we can inject configuration, but for SMTP authentication parameters.
```

Comment:
I would remove all this talk about PRs and add a section about what version is required to the top like in: https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos/credentials

Reply:
On it

Reply:
hope the new commit is what you had in mind